compiler/src/Language/Bond/Codegen/Cs/Types_cs.hs

```diff
@@ -111,7 +115,61 @@ namespace #{csNamespace}
         }|]
           where
             noCtor = not callBaseCtor && (fieldMapping == PublicFields && noMetaFields || null structFields)
-            noMetaFields = not $ getAny $ F.foldMap metaField structFields
+            callBaseCtor = getAny $ optional (foldMapFields metaField) structBase
+            baseCtor = if not callBaseCtor
+                then mempty
+                else [lt|
+            : base(fullName, name)|]
+
+        -- constructor: ConstructorParameters option
+        constructorWithParameters = if not noMetaFields
+            then error "bond_meta usage is incompatible with constructor_parameters"
+            else [lt|
+
+        public #{declName}(
+#{baseParameterBlock}            #{commaLineSep 3 thisParam structFields}
```

Comment:
It would more readable if you aligned the `#(baseParameterBlock}` here and not have the space padding it its definition.